crates/aptos/src/ascii/mod.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::common::types::CliResult;
+use clap::Subcommand;
+use indoc::indoc;
+
+/// Aptos Ascii Art commands
+///
+/// These commands is used to celebrate Aptos in spectacular ascii
+/// fashion!
+#[derive(Debug, Subcommand)]
+pub enum AsciiTool {
+    /// Print the Aptos logo in ascii
+    Logo,
+}
+
+impl AsciiTool {
+    pub async fn execute(self) -> CliResult {
+        match self {
+            AsciiTool::Logo => self.logo(),
+        }
+    }
+
```

Comment:
This trait is consistent across all the commands and ensures logging etc is automatically handled https://github.com/aptos-labs/aptos-core/blob/main/crates/aptos/src/account/create.rs#L28-L41